e2e/pages/adf/viewerPage.ts

```diff
@@ -62,12 +62,16 @@ export class ViewerPage {
     activeTab = element(by.css('div[class*="mat-tab-label-active"]'));
     toolbarSwitch = element(by.id('adf-switch-toolbar'));
     toolbar = element(by.id('adf-viewer-toolbar'));
+    lastButton = element.all(by.css('#adf-viewer-toolbar mat-toolbar > button[data-automation-id*="adf-toolbar-"]')).last();
```

Comment:
It's not enough to use just: button[data-automation-id*="adf-toolbar-"] ?

Reply:
This is the best locator I could find to get the list of buttons in the Viewer toolbar. But open to suggestions.